pkgs/development/python-modules/python-telegram-bot/default.nix

```
@@ -7,21 +7,22 @@
 , urllib3
 , tornado
 , pytest
+, APScheduler
 , isPy3k
 }:
 
 buildPythonPackage rec {
   pname = "python-telegram-bot";
-  version = "13.0";
+  version = "13.3";
   disabled = !isPy3k;
```

There is a [restriction](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/setup.py#L107) on the supported Python 3 release.

```suggestion
  disabled = pythonOlder "3.6";
```

---

We don't ship python 3 older than 3.6

---

Not sure how we handle this but I have no problem with adding disableds.